proposed/http-middleware/middleware.md

```diff
@@ -73,34 +72,34 @@ interface MiddlewareInterface
 {
     /**
      * Process an incoming server request and return a response, optionally delegating
-     * to the next middleware component to create the response.
+     * response creation to a handler.
      *
      * @param ServerRequestInterface $request
-     * @param DelegateInterface $delegate
+     * @param HandlerInterface $handler
      *
      * @return ResponseInterface
      */
     public function process(
         ServerRequestInterface $request,
-        DelegateInterface $delegate
+        HandlerInterface $handler
```

Comment:
With that new signature it is now even less clear why a handler is injected here. All this time I thought the new signature would be:

```php
function process(ServerRequestInterface $request, HandlerInterface $delegate)
```

How about this signature:

```php
function process(ServerRequestInterface $request, HandlerInterface $next)
```

?

That way we come back to what everyone is familiar with, and `next` is very clear to what it means (it's the next handler in the stack).